README.md

```diff
@@ -1,2 +1,107 @@
 # module-go-mysql
 ThingsDB module for communication with MySQL
+
+## Building the module
+
+To build the MySQL module, make sure Go is installed and configured.
+
+First go to the module path and run the following command to install the module dependencies:
+
+```
+go mod tidy
+```
+
+Next, you are ready to build the module:
+
+```
+go build
+```
+
+Copy the created binary file to the ThingsDB module path.
+
+## Configure the module
+
+The MySQL module must be configured before it can be used.
+
+In this example we will name the module `MySQL`. This name is arbitrary and can be anything you like. The example uses a database named `dbtest` with the
+default username and password combination.
+
+Run the following in the `@thingsdb` scope:
+
+```
+// The values MUST be change according to your situation, this is just an example
+new_module('MySQL', 'module-go-mysql', {
+    dsn: "anja:pass@/test",
+    conn_max_lifetime: 3,
+    max_idle_conn: 10,
+    max_open_conn: 1
+});
+```
+
+### Arguments
+
+Argument | Type | Description
+-------- | ---- | -----------
+`dsn` | `string` (optional) |
```

Comment:
I think the dsn is not optional, the Data Source Name (DSN) is probably required?

Reply:
Oops, yes that should be `required` .